dj/sql/functions.py

```diff
@@ -1206,15 +1206,18 @@ def infer_type(  # noqa: F811  # pragma: no cover
     return ct.ListType(element_type=arg.type)  # pragma: no cover
 
 
-class ArrayContains(Function):  # pragma: no cover  # pylint: disable=abstract-method
+class Array_Contains(
```

Comment:
@samredai actually we already handle the snake case to underscores with this function at the end of `functions.py`:
```
function_registry = FunctionRegistryDict()
for cls in Function.__subclasses__():
    snake_cased = re.sub(r"(?<!^)(?=[A-Z])", "_", cls.__name__)
    function_registry[cls.__name__.upper()] = cls
    function_registry[snake_cased.upper()] = cls
```
I think we can either keep this logic and keep the class name `ArrayContains` or switch all functions class names match the function name exactly. The former looks nicer from a Python class naming perspective I think.

Reply:
Oh nice! I must have just confused myself while I was running tests. Definitely think we should stick to the pythonic naming so I'll change this back.

Reply:
Alright, fixed 👍